Update VPAT July 2025 (#92)
PrairieLearn/marketing: src/pages/accessibility/index.mdx

| [2.4.1 Bypass Blocks](http://www.w3.org/TR/WCAG20/#navigation-mechanisms-skip) (Level A)                                    | Supports           | "Skip to content" links are available on all pages.                                                                                                                        |
| [2.4.2 Page Titled](http://www.w3.org/TR/WCAG20/#navigation-mechanisms-title) (Level A)                                     | Supports           | Pages include appropriate titles that describe their purpose.                                                                                                              |
| [2.4.3 Focus Order](http://www.w3.org/TR/WCAG20/#navigation-mechanisms-focus-order) (Level A)                               | Supports           | Focusable elements generally receive focus in top-to-bottom, left-to-right order.                                                                                          |

> | [2.4.4 Link Purpose (In Context)](http://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs) (Level A)                        | Supports           | Links fully identify the purpose of the links.                                                                   |

Reviewer: Wording nit:

```suggestion
| [2.4.4 Link Purpose (In Context)](http://www.w3.org/TR/WCAG20/#navigation-mechanisms-refs) (Level A)                        | Supports           | Links clearly convey their purpose within the surrounding context.                                               |
```

This will need Prettier if you want to accept it.
